huggle/mainwindow.cpp

```diff
@@ -3360,3 +3361,21 @@ void Huggle::MainWindow::on_actionShow_score_debug_triggered()
 
     HUGGLE_DEBUG1(debug_info);
 }
+
+void Huggle::MainWindow::on_actionPost_a_custom_message_triggered()
+{
+    if (!this->CheckExit() || !this->CheckEditableBrowserPage())
+        return;
+    if(this->CurrentEdit == nullptr)
+    {
+        Syslog::HuggleLogs->ErrorLog(_l("custommessage-none"));
+        return;
+    }
+    if (this->fCustomMessageForm != nullptr)
+        delete this->fCustomMessageForm;
```

Comment:
You also need to add `delete` to destructor of mainwindow class, other solution would be to not use this kind of pointer at all, instead you can add this statement to constructor of custom message form:
```
this->setAttribute(Qt::WidgetAttribute::WA_DeleteOnClose);
```
That way it will delete automatically on close and it would also support having multiple instances of this form open (so that user could have multiple message editors open in same time).

Reply:
I also think it's safe to call delete on nullptr, so that if statement is redundant.